server/src/service.rs

```diff
@@ -309,15 +309,18 @@ impl Service {
             .allow_name_replacements(true)
             .replace_existing_names(request_replacement)
             .name(oo7::dbus::api::Service::DESTINATION.as_deref().unwrap())?
-            .serve_at(
-                oo7::dbus::api::Service::PATH.as_deref().unwrap(),
-                service.clone(),
-            )?
             .build()
             .await?;
 
         service.connection.set(connection.clone()).unwrap();
+
         let object_server = connection.object_server();
+        object_server
+            .at(
+                oo7::dbus::api::Service::PATH.as_deref().unwrap(),
+                service.clone(),
+            )
```

Comment:
There is a difference in behaviour though, serve_at when called the moment of building the connection, will replace the existing instance of the served interface. Where ObjectServer::at doesn't. So basically the request_replacement no longer does what it is supposed to do.